Sources/BraintreePayPal/BTPayPalError.swift

```diff
@@ -114,6 +119,8 @@ public enum BTPayPalError: Error, CustomNSError, LocalizedError, Equatable {
             return "Missing BA Token for PayPal App Switch."
         case .missingPayPalRequest:
             return "The PayPal Request was missing or invalid."
+        case .failedToCreateEncodable:
+            return "Unable to create Encodable object"
```

Comment:
This may make it a bit more clear, but take it or leave it:
```suggestion
            return "The request was not a Checkout or Vault request."
```

Reply:
Based on the latest changes, we will no longer need to add this error